python/darpinstances/instance.py

```diff
@@ -283,6 +296,7 @@ def read_instance(filepath: Path, travel_time_provider: MatrixTravelTimeProvider
         line_string = infile.readline()
         action_id = 0
         index = 0
+        travel_time_divider = instance_config.get('travel_time_divider', 1)
```

Comment:
it has to work even without this property, have you tested it?

Reply:
Yes. The default value is 1 if it is not present in the config file.